http/watcher.go

```diff
@@ -74,11 +76,19 @@ func (w *BTCAddressWatcher) performCallForUser() error {
 		From:     q.LiquidityProviderRskAddress,
 		Signer:   w.lp.SignTx,
 	}
-
-	_, err = w.rsk.CallForUser(opt, q)
+	tx, err := w.rsk.CallForUser(opt, q)
 	if err != nil {
 		return err
 	}
+	ctx, cancel := context.WithTimeout(context.Background(), connectors.EthTimeout)
+	defer cancel()
+	s, err := w.rsk.GetTxStatus(ctx, tx)
+	if err != nil {
+		return nil
+	}
```

Comment:
If err != nil we should probably call GetStatus again, but not retry the transaction because it could have been mined anyways.

Reply:
I increased the timeout so we wait virtually forever for the getStatus to return something.